src/main/java/frc/utils/digitalinput/chooser/ChooserDigitalInput.java

```diff
@@ -0,0 +1,30 @@
+package frc.utils.digitalinput.chooser;
+
+import edu.wpi.first.wpilibj.smartdashboard.SendableChooser;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import frc.utils.digitalinput.DigitalInputInputsAutoLogged;
+import frc.utils.digitalinput.IDigitalInput;
+
+public class ChooserDigitalInput implements IDigitalInput {
+
+	private SendableChooser<Boolean> sendableChooser;
+
+	public ChooserDigitalInput(String chooserName) {
+		this.sendableChooser = new SendableChooser<>();
+		sendableChooser.setDefaultOption(
+			String.valueOf(ChooserDigitalInputConstants.DEFAULT_STATE),
+			ChooserDigitalInputConstants.DEFAULT_STATE
+		);
+		sendableChooser.addOption("true", true);
+		sendableChooser.addOption("false", false);
```

Comment:
do you need  defalut and add? i think u need just to setDefualt and add the other one

Reply:
but the DEFULT_STATE is in the constants, I don't necessarily know which one to not add